src/search/search_tables.h

```diff
@@ -109,10 +111,21 @@ class SearchTables {
         m_historyMoves.update(board, move, ply);
     }
 
+    inline movegen::Move getCounterMove(const movegen::Move& prevMove) const
+    {
+        return m_counterMoves.get(prevMove);
+    }
+
+    inline void updateCounterMoves(const movegen::Move& prevMove, const movegen::Move& counterMove)
```

Comment:
copy is cheaper (16 bits)
```suggestion
    inline void updateCounterMoves(movegen::Move prevMove, movegen::Move counterMove)
```